src/Dynamo.All.sln

```diff
@@ -905,6 +907,14 @@ Global
 		{133FC760-5699-46D9-BEA6-E816B5F01016}.Release|Any CPU.Build.0 = Release|Any CPU
 		{133FC760-5699-46D9-BEA6-E816B5F01016}.Release|x64.ActiveCfg = Release|Any CPU
 		{133FC760-5699-46D9-BEA6-E816B5F01016}.Release|x64.Build.0 = Release|Any CPU
+		{94CF053D-BB66-4FC7-883B-48F072701BA9}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{94CF053D-BB66-4FC7-883B-48F072701BA9}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{94CF053D-BB66-4FC7-883B-48F072701BA9}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{94CF053D-BB66-4FC7-883B-48F072701BA9}.Debug|x64.Build.0 = Debug|Any CPU
+		{94CF053D-BB66-4FC7-883B-48F072701BA9}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{94CF053D-BB66-4FC7-883B-48F072701BA9}.Release|Any CPU.Build.0 = Release|Any CPU
+		{94CF053D-BB66-4FC7-883B-48F072701BA9}.Release|x64.ActiveCfg = Release|Any CPU
+		{94CF053D-BB66-4FC7-883B-48F072701BA9}.Release|x64.Build.0 = Release|Any CPU
```

Comment:
do we want to build this tool by default ....with the rest of the solution  ?

Reply:
We do so for the other CLI tools, why wouldn't we?

Reply:
I guess if the tool does not directly depend on any other Dynamo projects it may have a slower dev cycle than the rest.
But I guess it is a small project (few cs files) so it does not really slow down the build process that much